zcutil/build.sh

```diff
@@ -1,7 +1,30 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
The argument for us is that it is far more likely that `/usr/bin/env` exists than `/bin/bash`, precisely because many projects use the env trick. This is kind of a similar issue to what we had with #638 / #1053, or #1841.

The argument-passing issue mentioned in the accepted answer there doesn't affect us, since it relates to passing arguments to the script interpreter, not into the script itself (that is done with `"$@"` further down in the script, which is left unchanged).